hal/ia32/exceptions.c

```diff
@@ -94,11 +94,13 @@ void *hal_exceptionsFaultAddr(unsigned int n, exc_context_t *ctx)
 {
 	u32 cr2;
 
-	__asm__ volatile
-	("movl %%cr2, %0"
-	:"=r" (cr2)
+	/* clang-format off */
+	__asm__ volatile (
+		"movl %%cr2, %0"
+	: "=r" (cr2)
 	:
-	:"eax");
+	: "eax");
```

Comment:
`eax` register is not used
```suggestion
	: );
```